### ISSUE TITLE
Crawler prunes anchor content from "content" results

### ISSUE THREAD

**Issue by jdeathe** [codelibs/fess#1262]
Since 11.2 and the addition of `a[rel="nofollow"]` to the setting `crawler.document.html.pruned.tags` in the file `fess_config.properties` the content being indexed has anchor content pruned.

Content example:
```
<p>Contact us using our <a href="https://www.domain.com/contact">contact form</a> or if you're visiting, you can <a href="//www.domain.com/location">get directions</a> to our head office.</p>
```
Is getting indexed as:
```
Contact us using our or if you're visiting, you can get directions to our head office.
```

Using the setting from the test case works as expected:
```
crawler.document.html.pruned.tags=noscript,script,style,header,footer,nav,a[rel=nofollow]
```

Ref
- https://github.com/codelibs/fess/commit/8e987df540872cad34b169d9b949831e35da3eb3#diff-eff61ee4803e2a1fad464fea66f4a810R122
- https://github.com/codelibs/fess/commit/8e987df540872cad34b169d9b949831e35da3eb3#diff-6a2c7a9b0d9b82a012d5c03fb1ca8ca9R134

**Comment by marevol** [comment]
Oh... Thank you for the info.
Fixed in #1263.

**Comment by jdeathe** [comment]
Thanks @marevol :)
